api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -19,57 +19,56 @@ paths:
     get:
       description: Signals list endpoint
       parameters:
-        - name: "created_before"
+        - name: id
           in: query
           description: >-
-            Filter returns only signals created before specified datetime. Note
-            the date time string should be in ISO 8601 format and URL encoded.
+            Filter Signals on a given id
           schema:
-            type: string
-            format: date-time
+            type: integer
+            example: 123456789
           required: false
-        - name: "created_after"
+        - name: address_text
           in: query
           description: >-
-            Filter returns only signals created after specified datetime. Note
-            the date time string should be in ISO 8601 format and URL encoded.
+            Filter Signals located on a (part) of an address (case insensitive)
           schema:
             type: string
-            format: date-time
+            example: 'Amstel 1'
           required: false
-        - name: "updated_before"
+        - name: area_code
           in: query
           description: >-
-            Filter returns only signals updated before specified datetime. Note
-            the date time string should be in ISO 8601 format and URL encoded.
+            Filter signals according to the currently assigned location area_code, accepts
+            a single area_code. area_type_code is required if area_code is present.
```

Comment:
Second sentence is no longer true, see: https://github.com/Amsterdam/signals/commit/6f408bf830b46d8e524e65093a95ecbaaf2d0995#diff-75a59f4ed16823ba3bedd159aeede8f01d502862d785d52024da5fcaeab43b8b